app/src/main/res/values-fr/strings.xml

```diff
@@ -169,4 +170,13 @@
     <string name="accountEditScreenButton">Editer compte</string>
 
 
+    <!-- Polls and Voting -->
+    <string name="polls_topbar_title">Votes et sondages</string>
+    <string name="polls_legend_text">Sondages de l\'événement</string>
+    <string name="polls_empty_poll_list">Il y aucun sondage pour cet événement</string>

```

Comment:
"Il n'y a aucun sondage pour cet événement" instead of "Il y a aucun sondage pour cet événement"

Reply:
Yes, my mistake. I will change that.